src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,310 @@
+---
+tags:
+- libp2p
+- Ethereum
+- Merge
+title: libp2p & Ethereum (the Merge)
+description:
+date: 2023-01-01
+permalink: ""
+translationKey: ''
+header_image: /libp2p_Ethereum_header.png
+author: Prithvi Shahi
+---
+
+If you've kept up with developments in Web3, you've likely heard of a significant event that occurred a few months ago: the [Paris](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md) upgrade, more widely known as [The Merge](https://ethereum.org/en/upgrades/merge/), on the [Ethereum network](https://ethereum.org/en/).
+This was a historic event for Ethereum, and one that was years in the making ([Google even had a countdown](https://decrypt.co/109428/google-counts-down-to-ethereum-merge-with-search-engine-easter-egg) for it.)
+
+You may be wondering: why is the libp2p blog writing about Ethereum and the Merge?
+Well, as a result of the Merge, we're excited to share that:
+**libp2p is integrated into the Ethereum network! :tada:**
+
+Today we'd like to highlight [years long effort behind this integration](#how-libp2p-was-integrated-into-ethereum), describe [how Ethereum uses libp2p](#how-libp2p-is-used-by-ethereum-validators-🔍), and celebrate this win for Ethereum and libp2p!
+
+# A Brief History of libp2p :hourglass_flowing_sand:
+
+Let's begin with a brief reflection on libp2p's history.
+
+[Protocol Labs](https://protocol.ai/) first developed [libp2p](https://libp2p.io/) as a networking library inside of [IPFS](https://ipfs.tech/).
+At the outset, their code and repositories were coupled. However, libp2p's potential and utility beyond IPFS was soon realized, and project maintainers split the two codebases apart. This enabled libp2p to truly become a product agnostic, reusable, general purpose networking library.
```

Comment:
Should we take more credit for this? "We realized the potential...", "we split the codebases".